libs/designsystem/src/lib/components/list/list.component.scss

```diff
@@ -158,6 +159,12 @@ ion-item-options[side='end'] {
   }
 }
 
+:host-context(.has-stand-alone) {
+  .list-items {
+    margin-bottom: 24px;
```

Comment:
We should use design tokens here instead of hardcoded value. See other places where we use `utils.size(...)`. `$sizes` map can be found in `libs/core/src/scss/base/_variables.scss`

Reply:
It actually just occurred to me that this styling is redundant, because I made an Input to control the margin. So I will just remove it. And I will keep in mind that we should use design tokens! :)